wolfcrypt/test/test.c

```diff
@@ -34154,10 +34174,10 @@ static int eccsi_imp_exp_pubkey_test(EccsiKey* key1, EccsiKey* key2)
     return 0;
 }
 
-static int eccsi_make_key_test(EccsiKey* priv, EccsiKey* pub, WC_RNG* rng,
+static wc_test_ret_t eccsi_make_key_test(EccsiKey* priv, EccsiKey* pub, WC_RNG* rng,
                                mp_int* ssk, ecc_point* pvt)
 {
-    int ret;
+    wc_test_ret_t ret;
```

Comment:
Is retyping the test result here (and all the other spots) going to have issues with wolfCrypt functions returning an `int` getting stored in the `wc_test_ret_t`? For example, here and the call to `wc_MakeEccsiKey()` a few lines down?

Reply:
Nope, `wc_test_ret_t` is expressly an `sword32` by default, and on a hypothetical target where `int` is bigger than `sword32`, `wc_test_ret_t` can be overridden to be an `sword64`.   The code compiles cleanly with that (that's how I tested it).

Btw this whole rigmarole arises from @lealem47 's current work an embedded system with a 16 bit `int`.  The test retval encoding scheme needs 32 bits, hence this.